common/taskana-common/src/main/java/pro/taskana/common/internal/configuration/DbSchemaCreator.java

```diff
@@ -64,9 +64,9 @@ public boolean run() throws SQLException {
     }
     if (LOGGER.isDebugEnabled()) {
       LOGGER.debug(outWriter.toString());
-    }
-    if (!errorWriter.toString().trim().isEmpty()) {
-      LOGGER.error(errorWriter.toString());
+      if (!errorWriter.toString().trim().isEmpty()) {
```

Comment:
Why is this only relevant when debugging?

Reply:
You are right, I will change it back